libraries/Microsoft.Bot.Builder.AI.QnA/Utils/GenerateAnswerUtils.cs

```diff
@@ -206,18 +215,26 @@ private async Task<QueryResults> QueryQnaServiceAsync(Activity messageActivity,
 
         private async Task EmitTraceInfoAsync(ITurnContext turnContext, Activity messageActivity, QueryResult[] result, QnAMakerOptions options)
         {
+            AnswerSpanRequest answerSpanRequest = null;
```

Comment:
Why would this ever be initialized to null? Null programming should be avoided when possible.

Reply:
Updated.

---
In reply to: [430556360](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r430556360) [](ancestors = 430556360)